V2_MIGRATION_GUIDE.md

```diff
@@ -60,164 +63,129 @@ Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequ
+- `SignupRequest` now requires the second parameter to be an `AuthenticationRequest`.
+- `ProfileRequest` now requires an `AuthenticationRequest` and a `Request<UserProfile, AuthenticationException>`.
 
-- `public AuthenticationAPIClient(@NonNull Context context)`. Use `public AuthenticationAPIClient(new Auth0(context))`.
+### Methods removed or changed
 
-#### UsersAPIClient
+#### `Auth0` removals and changes
 
-- `public UsersAPIClient(@NonNull Context context, @NonNull String token)`. Use `public UsersAPIClient(new Auth0(context), token)`.
+- `setOIDCConformant(boolean enabled)` has been removed. The SDK now only supports OIDC-Conformant applications.
+- `isOIDCConformant()` has been removed. The SDK now only supports OIDC-Conformant applications.
+- `doNotSendTelemetry()` has been removed. There is no replacement.
 
-#### SignupRequest
+#### `AuthenticationAPIClient` removals and changes
 
-- `public SignUpRequest(@NonNull DatabaseConnectionRequest<DatabaseUser, AuthenticationException> signUpRequest, @NonNull AuthRequest authRequest)`. Use `public SignUpRequest(@NonNull DatabaseConnectionRequest<DatabaseUser, AuthenticationException> signUpRequest, @NonNull AuthenticationRequest authenticationRequest)` instead.
+Methods and classes specific to calling any [Authentication APIs](https://auth0.com/docs/api/authentication) categorized as Legacy have been removed in v2. The following methods have been removed:
 
-#### ProfileRequest
+- `delegation()`. Support for Legacy Authentication API endpoints have been removed.
+- `delegationWithIdToken("{ID-TOKEN}")`. Support for Legacy Authentication API endpoints have been removed.
+- `delegationWithRefreshToken("{REFRESH-TOKEN}")`. Support for Legacy Authentication API endpoints have been removed.
+- `delegationWithIdToken("{ID-TOKEN}", "{API-TYPE}}")`. Support for Legacy Authentication API endpoints have been removed.
+- `loginWithOAuthAccessToken("{TOKEN}", {CONNECTION})`. For selected social providers, you can use `loginWithNativeSocialToken("{TOKEN}", "{TOKEN-TYPE}")` instead.
+- `tokenInfo("{ID-TOKEN}")`. Use `userInfo("{ACCESS-TOKEN}")` instead.
 
-- `public ProfileRequest(@NonNull AuthenticationRequest authenticationRequest, @NonNull ParameterizableRequest<UserProfile, AuthenticationException> userInfoRequest)`. Use `public ProfileRequest(@NonNull AuthenticationRequest authenticationRequest, @NonNull Request<UserProfile, AuthenticationException> userInfoRequest)` instead
-- `public ProfileRequest(@NonNull AuthRequest authRequest, @NonNull ParameterizableRequest<UserProfile, AuthenticationException> userInfoRequest)`. Use `public ProfileRequest(@NonNull AuthenticationRequest authRequest, @NonNull Request<UserProfile, AuthenticationException> userInfoRequest)` instead.
+Methods that returned a `ParameterizableRequest` now return a `Request`:
```

Comment:
This line was already called out at the beginning (before the details). Is it worth mentioning it again and listing every method that changed the return type? (Applies to other places as well)

Reply:
To me it makes sense to enumerate the specific changes somewhere.